### DISCUSSION TITLE
SoundSwitch didnt remember Settings after reboot OS

### DISCUSSION THREAD

**Discussion by teddysjam** [Belphemur/SoundSwitch#1615] [Q&A]
Hi, is there a way to keep my settings static?
After rebootings my OS i always have to set new my SteelSeries Headphone and Mic
![github](https://github.com/user-attachments/assets/93b39f14-85b7-41ae-86cf-f1ab7f0eb26c)

**Comment by ModalSeoul7** [comment]
I am also having this issue with the SteelSeries Artics Nova 7P.
